src/kde/kde.t.hh

```diff
@@ -0,0 +1,145 @@
+//--------------------------------------------*-C++-*---------------------------------------------//
+/*!
+ * \file   kde/kde.t.hh
+ * \author Mathew Cleveland
+ * \date   November 10th 2020
+ * \brief  Templated KDE functions for various dimensions and coordinate
+ * systems
+ * \note   Copyright (C) 2018-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//------------------------------------------------------------------------------------------------//
+
+#ifndef kde_kde_t_hh
+#define kde_kde_t_hh
+
+#include "kde.hh"
+#include <iostream>
+#include <math.h>
+#include <numeric>
+
+namespace kde {
+
+/*!
+ * reconstruction 
+ * \brief
+ *
+ * Cartesian geometry reconstruction of a 1D distribution
+ *
+ * \param[in] distribution 
```

Comment:
I don't know a lot about KDEs so it would help me to have some of this part filled out (I'll ask my questions in mattermost for now)